infra/buckets/s3.tf

```diff
@@ -150,3 +150,12 @@ resource aws_s3_bucket dss_s3_checkout_bucket_unwritable {
 }
 POLICY
 }
+
+resource aws_s3_bucket dss_s3_events_bucket {
+  count = "${length(var.DSS_FLASHFLOOD_BUCKET) > 0 ? 1 : 0}"
+  bucket = "${var.DSS_FLASHFLOOD_BUCKET}"
+  server_side_encryption_configuration {
+    rule {apply_server_side_encryption_by_default {sse_algorithm = "AES256"}}
+  }
+  tags = "${local.common_tags}"
```

Comment:
Is there a reason this isn't `tags = "${merge(local.common_tags, local.aws_tags)}"`?

Reply:
fixed